### PR TITLE
tickets/DM-38551: Fix assignment of blend_centroid_x and blend_centroid_y in donut catalogs.

### PR DESCRIPTION
**Pull request by jbkalmbach**

### CODE REVIEW

File: python/lsst/ts/wep/task/generateDonutCatalogWcsTask.py [hunk 1]
```diff
@@ -291,6 +291,8 @@ def donutCatalogToDataFrame(
         fieldObjects["blend_centroid_x"] = blendCX
         fieldObjects["blend_centroid_y"] = blendCY
 
+        print(fieldObjects)
```
Comment (teweitsai) [inline python/lsst/ts/wep/task/generateDonutCatalogWcsTask.py]: Do you still need this? Or use the `log` instead?

File: tests/task/test_generateDonutCatalogWcsTask.py [hunk 2]
```diff
@@ -198,6 +198,67 @@ def testDonutCatalogToDataFrame(self):
...
+        )
+
+        blendXVals = [
+            donutCatSmall['centroid_x'][0] + 10.,
+            donutCatSmall['centroid_x'][1] + 15.
+        ]
+        blendYVals = [
+            donutCatSmall['centroid_y'][0] + 10.,
+            donutCatSmall['centroid_y'][1] + 15.
+        ]
```
Comment (teweitsai) [inline tests/task/test_generateDonutCatalogWcsTask.py]: Maybe `blendXVals` and `blendYVals` to be `blendValsX` and `blendValsY` instead?

**Review by teweitsai — APPROVED** [review]
Looks good. Thanks for this improvement!

File: python/lsst/ts/wep/task/generateDonutCatalogWcsTask.py (hunk 1 above)
Comment (jbkalmbach) [reply python/lsst/ts/wep/task/generateDonutCatalogWcsTask.py]: No, thanks for catching that.

**Pull request merged**